sources/util.ts

```diff
@@ -56,6 +56,16 @@ export function isHTMLLegendElement(
 	return isElement(node) && node.tagName === "LEGEND";
 }
 
+export function isHTMLSvgElement(node: Node | null): node is SVGElement {
+	return isElement(node) && node.tagName === "svg";
+}
+
+export function isHTMLSvgTitleElement(
+	node: Node | null
+): node is SVGTitleElement {
+	return isElement(node) && node.tagName === "title";
```

Comment:
```suggestion
	return isSVGElement(node) && node.tagName === "title";
```
Otherwise `<title>` in `<head>` is considered one as well

`isSVGElement` can be implemented by checking for existence of the `ownerSVGElement` property.